core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.html

```diff
@@ -165,7 +168,7 @@
       display: none;
     }
   .oppia-translate-text-header-container .oppia-opportunities-subheading {
-    color: #009688;
```

Comment:
Just to confirm, why are we making these changes and can you please share a before/after screenshot of the UI of this change? (This will help me understand the changes better and ensure I follow these color schemes while reviewing future PRs.)

Reply:
Yeah would be nice to have this. Then I would give my approval.